new-packages/Core/src/Operators/Raw.candy

```diff
@@ -1,105 +1,117 @@
-## Binary operators.
+## The Candy compiler knows the traits in this module and enables operators for types that implement
+## them.
 
-trait BinaryEqualEqual[Other, Result] {
-  # The binary `==` operator.
+## TODO(marcelgarus): Document precedence here.
 
-  fun equalEqual(other: Other): Result
+
+## Infix operators.
+##
+## These are operators that are placed between two operands. For example, `+` is an infix operator
+## because you can write `a + b` to apply `+` to `a` and `b`.
```

Comment:
This would be good documentation for new users. Maybe move operators into submodules `Infix` and `Prefix` and use these comments as doc comments?